frontend/src/core/user/components/FileDownload.js

```diff
@@ -0,0 +1,56 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+
+import type { NavigationParams } from 'core/navigation';
+
+
+type Props = {|
+    parameters: NavigationParams,
+|};
+
+
+/*
+ * Render a Sign Out link.
+ */
+export default class SignOut extends React.Component<Props> {
+    downloadTranslations = () => {
+        document.getElementById('download-form').submit();
+    }
+
+    render() {
+        const { parameters } = this.props;
+
+        let csrfToken = '';
+        const rootElt = document.getElementById('root');
+        if (rootElt) {
+            csrfToken = rootElt.dataset.csrfToken;
+        }
```

Comment:
We should factor this out and reuse in `core/api/base.js`.

Reply:
There's a `getCSRF` method in api.base maybe you can just use that? Or maybe that's what you meant and I'm being a fool? :clown_face: